backend/src/main/resources/application.properties

```diff
@@ -122,3 +122,7 @@ relations-api.authn.client.secret=development-value-123
 relations-api.authn.mode=oidc-client-credentials
 relations-api.is-secure-clients=false
 relations-api.target-url=${clowder.endpoints.kessel-relations-api:localhost:9000}
+
+# Enabling "trace" logging level for debugging purposes.
+quarkus.log.category."com.redhat.cloud.notifications.db.builder.Parameters".min-level=TRACE
+quarkus.log.category."com.redhat.cloud.notifications.routers.EndpointResource".min-level=TRACE
```

Comment:
Do you plan on reverting changes from this PR after the debugging is done?

Reply:
Yes, that was my idea. Let me leave a comment there just in case...